<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDialogData</class>
 <widget class="QDialog" name="TDialogData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>工号</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBoxEmpNo">
          <property name="maximum">
           <number>990000</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditName">
          <property name="text">
           <string>李明</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>性别</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxSex">
          <item>
           <property name="text">
            <string>男</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>女</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>出生日期</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDateTimeEdit" name="dateTimeEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>出生省份</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxBirth">
          <item>
           <property name="text">
            <string>河北</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>湖南</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>湖北</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>安徽</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>河南</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>上海</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>重庆</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>部门</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="comboBoxDepartment">
          <item>
           <property name="text">
            <string>技术部</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>生产部</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>销售部</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>行政部</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>工资</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="spinBoxSalary">
          <property name="maximum">
           <number>99000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>照片</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelpic">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonImport">
        <property name="text">
         <string>导入照片</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/images/00.JPG</normaloff>:/images/images/00.JPG</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClear">
        <property name="text">
         <string>清除照片</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/images/103.bmp</normaloff>:/images/images/103.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOK">
        <property name="text">
         <string>确定</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/images/704.bmp</normaloff>:/images/images/704.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCANCEL">
        <property name="text">
         <string>取消</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/images/706.bmp</normaloff>:/images/images/706.bmp</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>备注</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>203</width>
       <height>106</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>TDialogData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCANCEL</sender>
   <signal>clicked()</signal>
   <receiver>TDialogData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
